src/pages/Menu.js

```diff
@@ -1,7 +1,117 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
may be this file should use name of MenuTabs; setup is quite style intensive

Reply:
Agreed. I've changed it to MenuTabs